### DISCUSSION TITLE
The rendering of the image is blurry and unclear, but the photo opened with the photo software of Windows system is clear. What is the reason

### DISCUSSION THREAD

**Discussion by Haidiye01** [google/filament#7381] [General]
![img_v3_025f_e08300f6-1471-48b4-aa4c-5828e85c9e4g](https://github.com/google/filament/assets/40328051/44799e51-aa6e-4716-9d42-0c02e89d070c)
On the left is the photo software for Windows, and on the right is the rendering effect
The rendering of the image is blurry and unclear, but the photo opened with the photo software of Windows system is clear. What is the reaso？
Thanks

**Comment by romainguy** [comment]
How do you render that image?
